### PR TITLE
Fix stepper directions

### PR DESCRIPTION
**Pull request by andrewbecker**
Reversed x, y, and extruder directions based off of the kit.

### CODE REVIEW

**Review by lostapathy — CHANGES_REQUESTED** [review]
If we're updating the variable on these, we should also update the comment to match so that they stay in sync.

File: Duet/ProjectR3D/300ZL/config.g [hunk 1]
```diff
@@ -18,10 +18,10 @@ M83                                 	; ...but relative extruder moves
 M667 S1								; CoreXY mode
 
 M584 X0 Y1 Z5:6:7 E3:4:8:9 		; Map Z to drivers 5, 6, 7. Define unused drivers 3,4,8 and 9 as extruders
-M569 P0 S1                          ; Drive 0 goes forwards (change to S0 to reverse it) X stepper
-M569 P1 S0                          ; Drive 1 goes backwards	Y Stepper
+M569 P0 S0                          ; Drive 0 goes forwards (change to S0 to reverse it) X stepper
```
Comment (natewalck) [inline Duet/ProjectR3D/300ZL/config.g]: There are many reasons why the steppers might go in a different direction (wiring, etc), so I think it is best to standardize on the Project R3d config.

See: https://github.com/railcore/configs/blob/master/Duet/ProjectR3D/300ZL/config.g#L21-L28

**Comment by natewalck** [comment]
I didn't realize this was to the project r3d config repo at first.

I assume you have a kit and the config as-provided was backwards?

**Comment by andrewbecker** [comment]
Yeah I have the kit and noticed mine were backwards. I also saw where Wade
from the Facebook group had to reverse his as well.

Andy

**Comment by natewalck** [comment]
Perfect. Thanks for cutting this PR!

**Comment by andrewbecker** [comment]
Thanks for fixing the comments for the stepper motors!

**Pull request merged**